shared/components/Animation/__tests__/Reveal.spec.jsx

```diff
@@ -24,3 +24,25 @@ describe('Reveal', () => {
     expect(reveal).toMatchSnapshot()
   })
 })
+
+describe('accessibility', () => {
+  it('marks the panel as an expandable trigger for content for screen readers', () => {
```

Comment:
This test description seems to be referring to `aria-expanded`...

Maybe it should read `it('shows panel content to screen readers when opened')`